src/cascadia/TerminalSettingsEditor/PreviewConnection.cpp

```diff
@@ -18,6 +18,9 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 
     void PreviewConnection::Start() noexcept
     {
+        // First send a sequence to disable cursor blinking
+        _TerminalOutputHandlers(L"\x1b[?12l");

```

Comment:
very clever. we do also remove the cursor when it loses focus, though, too.

Reply:
Ah, the preview control is no longer enabled nor focusable (see Profiles.cpp lines 484-485) - so the cursor just stays there and with this sequence it doesn't blink